modules/ROOT/pages/policies-policy-level.adoc

```diff
@@ -0,0 +1,116 @@
+= Policy Level
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies implement an additional level of granularity with SLA tiering. Policies control access based on the following access structuring:
+
+* Resource-level policies
+Impose a limit on all requests that match the resource to which it is applied. All policies, except the Cross-Origin Resource Sharing (CORS) policy, are resource-level.
```

Comment:
It doesn't impose a limit (behavior depends on the policy applied), it applies the policy on all requests that match.

Reply:
Done! Thanks!